src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -251,13 +252,13 @@ $chevron-width: $nav-card-horizontal-spacing;
   .leaf-link {
     color: var(--color-figure-gray-secondary);
     font-weight: $font-weight-semibold;
-    // groups dont need the overlay
-    &:after {
-      display: none;
-    }
   }
 }
 
+.link {
+  line-height: 1;
```

Comment:
```suggestion
  min-width: 0;
```

Reply:
Also here should be block - https://github.com/apple/swift-docc-render/pull/456/files#diff-0770d1c4ed499b9fc69b4a7e9e024e5bc5539eec417365e0364a394a5dc9d3b4R310

And I think we are good.